public.openapi.json

```diff
@@ -735,6 +735,16 @@
               "$ref": "#/components/schemas/DomainIpaServer"
             }
           },
+          "automount_locations": {
+            "description": "List of automount locations for AutoFS",
+            "type": "array",
+            "items": {
+              "type": "string",
+              "minLength": 1
+            },
+            "minItems": 0,
```

Comment:
To clarify, these lines mean that the `automount_locations` array has a minimum length of 0; and the *members* of that array are strings with minimum length 1?

That sounds fine to me, just checking that I am reading this right.

Reply:
That's correct.